perfkitbenchmarker/providers/openstack/flags.py

```diff
@@ -64,6 +64,10 @@
 flags.DEFINE_integer('openstack_volume_size', None,
                      'Size of the volume (GB)')
 
+flags.DEFINE_string('openstack_volume_type',
```

Comment:
Hi,

Thank you for the response. Please tell me if I have misunderstood, but it sounds like you mean that I should define some options for the volume type.

Due to the flexibility in OpenStack (Cinder) it is possible to name a 'volume type' as the cloud administrator sees fit, therefore I believe the user should have full control of what they input after this flag.

Thanks.